python/lsst/daf/butler/core/config.py

```diff
@@ -894,7 +894,7 @@ def updateParameters(configType, config, full, toUpdate=None, toCopy=None, overw
             for key in toCopy:
                 if key in localConfig and not overwrite:
                     log.debug("Not overriding key '%s' from defaults in config %s",
-                              key, localConfig.__class__.__name__)
+                              key, value, localConfig.__class__.__name__)
```

Comment:
The log message has to placeholders so you can not put three parameters in the call here (ie I think you were right the first time -- or you need to edit the log message).

Reply:
I removed value here, I can make a different PR if you want the changes made immediately. Or just make them yourself if that's easier and faster.